internal/controller/apiextensions/claim/reconciler.go

```diff
@@ -501,6 +502,19 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		if kerrors.IsConflict(err) {
 			return reconcile.Result{Requeue: true}, nil
 		}
+		if errors.Is(err, errBindCompositeConflict) {
```

Comment:
As long as `composite.Configurator` is a public interface and we handle this error here, the error must be public too and well documented as part of the interface.

Sidenote: am not convinced that `composite.Configurator` is of value as a public interface. Seems to be a very leaky abstraction.

Reply:
> As long as composite.Configurator is a public interface and we handle this error here, the error must be public too and well documented as part of the interface.

Done.

Reply:
:+1: